### PR TITLE
Channel Bug Fixes

### PR DESCRIPTION
**Pull request by sourabhmodi20051**
1. Refresh all components on update issue #93 
2. Refresh channel list on create new channel 
3. Fix join channel and leave channel issue #104

### CODE REVIEW

File: client/package.json [hunk 1]
```diff
@@ -8,6 +8,7 @@
     "classnames": "^2.2.6",
     "enzyme": "^3.10.0",
     "enzyme-adapter-react-16": "^1.14.0",
+    "lodash": "^4.17.15",
```
Comment (rakeshtembhurne) [inline client/package.json]: `lodash-es` has been merged in previous PRs. `lodash` library is loads complete file even if we are using a single function. This issue has been solved in `lodash-es` package.

File: client/src/containers/ChannelMembers/ChannelMembers.js [hunk 2]
```diff
@@ -1,75 +1,121 @@
...
+    trigger
+  );
+  const [members, setMembers] = useState([]);
+  const { isLoading } = channelDetails;
+  const { loginStatus } = useAppContext();
+  if (isLoading) {
+    return <Spinner />;
+  }
+  const memberIds = members.map(member => member._id);
+  console.log(uniq(memberIds).sort());
```
Comment (rakeshtembhurne) [inline client/src/containers/ChannelMembers/ChannelMembers.js]: Were `console.log` kept intentionally?

Comment (sourabhmodi20051) [reply client/src/containers/ChannelMembers/ChannelMembers.js]: removed this

File: client/package.json (hunk 1 above)
Comment (sourabhmodi20051) [reply client/package.json]: updated to lodash-es

File: client/src/containers/ChannelMembers/ChannelMembers.js [hunk 3]
```diff
@@ -7,8 +7,8 @@ import callApi from "../../libs/axios";
 import { fetchMembersData } from "../../containers/Workspace/utils";
 import { useAppContext } from "../../containers/App/AppContext";
 import AddMember from "./AddMember";
-import isEqual from "lodash/isEqual";
-import uniq from "lodash/uniq";
+import isEqual from "lodash-es/isEqual";
```
Comment (rakeshtembhurne) [inline client/src/containers/ChannelMembers/ChannelMembers.js]: You can import form single source like 
`import { isEqual, uniq } from "lodash-es"

**Pull request merged**